iis-core/src/main/java/eu/dnetlib/iis/core/common/JsonTestUtils.java

```diff
@@ -0,0 +1,51 @@
+package eu.dnetlib.iis.core.common;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.avro.generic.GenericRecord;
+import org.apache.commons.io.IOUtils;
+
+import com.google.common.collect.Lists;
+import com.google.gson.Gson;
+
+/**
+ * Utils class for operating on avro objects saved in json files
+ * 
+ * @author madryk
+ *
+ */
+public class JsonTestUtils {
+
+
+    //------------------------ CONSTRUCTORS --------------------------
+
+    private JsonTestUtils() {
+        throw new IllegalStateException("may not be instantiated");
+    }
+
+
+    //------------------------ LOGIC --------------------------
+
+    /**
+     * Reads avro objects from json file
```

Comment:
What the method really does is "Reads records from json file (jsonFilePath) as objects of recordClass class."